bin/nexus-repositories-create

```diff
@@ -43,28 +43,28 @@ function find_repo(){
     # If changes are needed, ensure that testing is done on both Nexus 3.25.0 (csm-1.2) and
     # 3.38.0 (csm-1.3) since this is used in both versions.
     if [[ $# -ne 3 ]]; then
-        echo -n >&2 "ERROR Missing arguments. Expecting repo_format repo_type repo_name. Arguments passed were: $@"
+        echo -n >&2 "Missing arguments. Expecting repo_format repo_type repo_name. Arguments passed were: $@"
         return 1
     fi
-    if [[ -z "$1" ]]; then echo -n >&2 "ERROR Empty repo_format argument"; return 1; fi
-    if [[ -z "$2" ]]; then echo -n >&2 "ERROR Empty repo_type argument"; return 1; fi
-    if [[ -z "$3" ]]; then echo -n >&2 "ERROR Empty repo_name argument"; return 1; fi
+    if [[ -z "$1" ]]; then echo -n >&2 "Empty repo_format argument."; return 1; fi
+    if [[ -z "$2" ]]; then echo -n >&2 "Empty repo_type argument."; return 1; fi
+    if [[ -z "$3" ]]; then echo -n >&2 "Empty repo_name argument."; return 1; fi
     repo_format=$1
     repo_type=$2
     repo_name=$3
 
-    echo -n >&2 "DEBUG Looking for ${repo_format}/${repo_type} repository: ${repo_name} ..."
+    echo -n >&2 "Looking for ${repo_format}/${repo_type} repository: ${repo_name}..."
 
     # Ask Nexus for all repositories and filter that result by format, type, and name.
     jqfilter=".[] | select(.format == \"$repo_format\") | select(.type == \"$repo_type\") | select(.name == \"$repo_name\")"
     result=$(curl -sfk "${URL}/v1/repositories" | jq "$jqfilter")
 
     # An empty string here means that the repo was not found in the json result by format, type, and name.
     if [[ -z "$result" ]]; then
-      echo -n >&2 "ERROR Could not find the ${repo_format}/${repo_type} repository: ${repo_name}"
+      echo -n >&2 " NOT found..."
```

Comment:
IUF will not be able to recognize this message if ERROR is removed. Additional information is for troubleshooting.